include/effects/matrix/PatternCube.h

```diff
@@ -80,20 +80,19 @@
 class PatternCube : public EffectWithId<PatternCube>
```

Comment:
Cube pattern was hardcoded in size before, now adapts to available canvas size and fills it with smaller cubes